CHANGELOG.md

```diff
@@ -5,25 +5,6 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [11.9.0] - 2023-03-XX
```

Comment:
Please revert your changes to this file.

Reply:
I got the git remotes messed. Now is untouched